src/commands/cmd_string.cc

```diff
@@ -86,6 +86,8 @@ class CommandGetEx : public Commander {
       uint32_t max_btos_size = static_cast<uint32_t>(config->max_bitmap_to_string_mb) * MiB;
       redis::Bitmap bitmap_db(svr->storage, conn->GetNamespace());
       s = bitmap_db.GetString(args_[1], max_btos_size, &value);
+      if (ttl_ > 0) bitmap_db.Expire(args_[1], ttl_ + util::GetTimeStampMS());
+      if (persist_) bitmap_db.Expire(args_[1], 0);
```

Comment:
It would be more consistent to use `if..else if` since it's exclusive between TTL and PERSIST.
```suggestion
      if (ttl_ > 0) {
        bitmap_db.Expire(args_[1], ttl_ + util::GetTimeStampMS());
      } else if (persist_) {
        bitmap_db.Expire(args_[1], 0);
      }
```